domains/ohmycv.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmnid@gmail.com"
+    },
+
+    "record": {
+        "URL": "https://ohmycv.netlify.app"
```

Comment:
I cannot reach this website.

Reply:
The website has been unpublished, please delete it ohmycv.json